openpype/hosts/blender/api/plugin.py

```diff
@@ -1396,25 +846,259 @@ def switch(
     def remove(self, container: Dict) -> bool:
         """Remove an existing container from a Blender scene.
 
+        As Blender purges datablocks is not used,clearing all
```

Comment:
```suggestion
        As Blender datablocks purge is not used, clearing all
```

```suggestion
        As datablocks are not purged in Blender, clearing all
```

Seems oddly worded, and a missing space after `,`.